service ListDevices {
  list<string> getDevices()
}
